addons/crm/tests/test_crm_lead.py

```diff
@@ -805,3 +806,8 @@ def test_phone_mobile_update(self):
         self.assertEqual(lead.phone, self.test_phone_data[1])
         self.assertEqual(lead.mobile, self.test_phone_data[2])
         self.assertFalse(lead.phone_sanitized)
+
+
+class TestLeadFormatAddress(FormatAddressCase):
+    def test_address_view(self):
+        self.assert_address_view('crm.lead')
```

Comment:
Side note: we generally use camelCase for asserts (assertAddressView) in our apps, to match the writing used in unit test framework of python. But that's not blocking obviously ^^ .